earth2mip/schema.py

```diff
@@ -360,6 +364,9 @@ class EnsembleRun(pydantic.BaseModel):
     output_dir: Optional[str] = None
     output_path: Optional[str] = None
     restart_frequency: Optional[int] = None
+    grf_noise_alpha: float = 2.0
+    grf_noise_sigma: float = 5.0
+    grf_noise_tau: float = 2.0
```

Comment:
@yairchn where are these parameters used in the code? I dont see any additional code that actually references them?